src/main/java/ru/ayaz/MessageParser.java

```diff
@@ -1,12 +1,12 @@
 package ru.ayaz;
 
-public class MessageProcessor {
+public class MessageParser {
```

Comment:
&#x1F34E; **Issue**
This class also looks like it became a victim of splitting logic between classes. But this one actually gathered method from 2 distinct Domain entities. You get rid of this class and move respective methods inside fitting Domain classes.